lib/ruby-debug-ide/xml_printer.rb

```diff
@@ -389,9 +392,19 @@ def inspect_with_allocation_control(slice, memory_limit)
       result = nil
       inspect_thread = DebugThread.start {
         start_alloc_size = ObjectSpace.memsize_of_all
```

Comment:
> Perhaps, it is worth running a :call and :c_call handler a little less often. Every fifth, for example.

That's what I'm talking about.

`inspect` often works fast, but
1. We add constant overhead on each call which may slow execution down in dozens of times;
2. There are many `inspects` during variables collection so in total this matters